### PR TITLE
Add --check option for verifying word_list correctness and checksum.

### PR DESCRIPTION
**Pull request by acsaba22**

### CODE REVIEW

File: bip39.py [hunk 1]
```diff
@@ -2,17 +2,43 @@
...
+    if not (len(lines) in [1, 2]):
+        print("wrong input")
+        sys.exit(1)
+
+    words = lines[0]
+
+    if (args.check):
+        if (passw):
+            print("Can't check when password is present")
+            sys.exit(1)
```
Comment (errge) [inline bip39.py]: This is possible to do, because password is hashed to the whole seed after checksum validation.

File: bip39.py [hunk 2]
```diff
@@ -2,17 +2,43 @@
...
+            print("Unkown words!")
+            sys.exit(1)
+        if not checksum_ok:
+            print("Checksum NOT OK!")
+            sys.exit(1)
+
+    print(util.bh2u(keystore.bip39_to_seed(words, passw)))
+
+
+if __name__== "__main__":
```
Comment (errge) [inline bip39.py]: lint

File: bip39.py [hunk 3]
```diff
@@ -2,17 +2,43 @@
...
+    words = lines[0]
+
+    if (args.check):
+        if (passw):
+            print("Can't check when password is present")
+            sys.exit(1)
+
+        (checksum_ok, wordlist_ok) = keystore.bip39_is_checksum_valid(words)
+        if not wordlist_ok:
+            print("Unkown words!")
```
Comment (errge) [inline bip39.py]: stderr

File: bip39.py [hunk 4]
```diff
@@ -2,17 +2,43 @@
...
+            sys.exit(1)
+        if not checksum_ok:
+            print("Checksum NOT OK!")
```
Comment (errge) [inline bip39.py]: stderr

File: bip39.py [hunk 5]
```diff
@@ -2,17 +2,43 @@
...
+    args = arg_parser.parse_args()
+
+    lines = [x.strip() for x in sys.stdin.readlines()]
+
+    words = ""
+    passw = ""
+    if len(lines) == 2:
+        passw = lines[1]
+    if not (len(lines) in [1, 2]):
+        print("wrong input")
```
Comment (errge) [inline bip39.py]: stderr

File: bip39.py [hunk 6]
```diff
@@ -2,17 +2,43 @@
 
 from electrum import util, keystore
 
+import argparse
```
Comment (errge) [inline bip39.py:5]: research: what's up with gflags

Comment (acsaba22) [reply bip39.py:5]: maybe later.

File: bip39.py (hunk 2 above)
Comment (acsaba22) [reply bip39.py]: done

File: bip39.py (hunk 4 above)
Comment (acsaba22) [reply bip39.py]: done

File: bip39.py (hunk 3 above)
Comment (acsaba22) [reply bip39.py]: done

File: bip39.py (hunk 1 above)
Comment (acsaba22) [reply bip39.py]: done.

File: bip39.py (hunk 5 above)
Comment (acsaba22) [reply bip39.py]: done

**Pull request merged**